Source/CesiumRuntime/Private/Tests/CesiumLoadTest.cpp

```diff
@@ -299,15 +298,93 @@ bool RunLoadTest(const size_t locationIndex) {
         loadElapsedTime);
   }
 
+  if (afterTest) {
+    afterTest(context);
+  }
```

Comment:
This `afterTest` thing allows doing additional checks after the performance test is complete. For example, to make sure the test did what you expected. Or in my case, to do an additional test after zooming out.

In retrospect, though, I think maybe it would be better to restructure things further so that these std::functions aren't needed at all. For example, the Denver test could instead look like this:

```cpp
LoadTestContext context;
createCommonWorldObjects(context);
setupForDenver(context);
RunLoadTest(contex);
```

Where `RunLoadTest` starts with "Halt tileset updates and reset them", skipping the context and object creation.

Reply:
Agreed. The `RunLoadTest ` part is really the test code we care about. The `setupForXXX` should be clearly separate and reusable. What if we want this setup code for other tests?